### PR TITLE
Fix a bunch of bugs, improve container logic

### PR DESCRIPTION
**Pull request by spaceisfun**

### CODE REVIEW

**Review by patelp3989 — APPROVED** [review]

**Review by patelp3989 — APPROVED** [review]

File: src/CoreLibrary/Resources/Resources.resx [hunk 1]
```diff
@@ -252,11 +252,11 @@
   <data name="ReadyToCommunicate" xml:space="preserve">
     <value>Factory Orchestrator Service is ready to communicate with client(s).</value>
   </data>
-  <data name="RedirectingToRunAsExplorerUser" xml:space="preserve">
-    <value>{0} is a Win32 GUI program. Redirecting to run via RunAsExplorerUser...</value>
+  <data name="RedirectingToRunAsRDUser" xml:space="preserve">
```
Comment (hajya) [inline src/CoreLibrary/Resources/Resources.resx:255]: does this have to be 'RD' user? i see the exes is called that, but is there a reason we don't just stick to a more abstract 'remote' user and then we can avoid having to change it 'by tech'.

**Review by hajya — COMMENTED** [review]
Took a first pass. Happy to finish tomorrow. :)

File: src/CoreLibrary/Resources/Resources.resx (hunk 1 above)
Comment (spaceisfun) [reply src/CoreLibrary/Resources/Resources.resx:255]: good idea, ill keep the OS exe named as is, but update here :)

**Comment by spaceisfun** [comment]
@microsoft/onecore submitting this so I can get it in official builds ASAP. Still appreciate reviews when you can.

**Pull request merged**